ci/generate-markdown/generate-packaging.sh

```diff
@@ -225,67 +225,40 @@ the case, the instructions describe how you can manually download and install
 these dependencies.
 END_OF_PREAMBLE
 
-readonly DOCKERFILES_DIR="${BINDIR}/../kokoro/install"
-
-echo
-echo "<details>"
-echo "<summary>Fedora (33)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.fedora"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>openSUSE (Leap)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.opensuse-leap"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>Ubuntu (20.04 LTS - Focal Fossa)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.ubuntu-focal"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>Ubuntu (18.04 LTS - Bionic Beaver)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.ubuntu-bionic"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>Ubuntu (16.04 LTS - Xenial Xerus)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.ubuntu-xenial"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>Debian (Buster)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.debian-buster"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>Debian (Stretch)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.debian-stretch"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>CentOS (8)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.centos-8"
-echo "</details>"
-
-echo
-echo "<details>"
-echo "<summary>CentOS (7)</summary>"
-echo "<br>"
-"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.centos-7"
-echo "</details>"
+# Extracts the part of a file between the START/END tags.
+function extract() {
+  sed -e '0,/^.*\[START packaging.md\].*$/d' \
+    -e '/^.*\[END packaging.md\].*$/,$d' "$1"
+}
+
+# A "map" (comma separated) of dockerfile -> summary.
+DOCKER_DISTROS=(
+  "demo-fedora.Dockerfile,Fedora (33)"
+  "demo-opensuse-leap.Dockerfile,openSUSE (Leap)"
+  "demo-ubuntu-focal.Dockerfile,Ubuntu (20.04 LTS - Focal Fossa)"
+  "demo-ubuntu-bionic.Dockerfile,Ubuntu (18.04 LTS - Xenial Xerus)"
+  "demo-ubuntu-xenial.Dockerfile,Ubuntu (16.04 LTS - Xenial Xerus)"
```

Comment:
16.04 is Xenial Xerus, 18.04 is Bionic Beaver:

https://wiki.ubuntu.com/Releases

Reply:
Oops! Fixed.